components/textbox/textbox.tsx

```diff
@@ -248,7 +248,7 @@ export default class Textbox extends React.PureComponent<Props> {
 
             preview = (
                 <div
-                    tabIndex={0}
+                    tabIndex={this.props.tabIndex || 0}
```

Comment:
Maybe?
```suggestion
                    tabIndex={this.props.tabIndex ?? 0}
```
to allow for `-1`

Reply:
Not sure that's necessary? `-1 || 0` in the console returns -1 for me (-1 isn't falsy) ... as does `-1 ?? 0`. Also not sure if there is a preferred approach here? I feel like this is fine as we want the provided value to be a legit number so any falsy values supplied will default to `0`. Of course, typescript should help out here so back at I'm not sure if one is preferred over the other.

Reply:
Ahhh, my bad! Makes sense to me now.